css-basics_social-card/css/styles.css

```diff
@@ -14,8 +14,8 @@ body {
 }
 
 .content {
-  width: 50%;
-  height: auto;
+  width: 95%;
```

Comment:
I think this property is not really needed, although it doesn't break the layout. You can just leave the original value and the body default padding will do the job.